docs/api.asciidoc

```diff
@@ -26,6 +26,18 @@ entirely.
 
 NOTE: You cannot access these APIs via the Console in Kibana.
 
+== Request Headers
+
+The following headers must be set on all non-whitelisted API endpoints 
+(See https://www.elastic.co/guide/en/kibana/current/kibana-authentication.html#saml[SAML Authentication]
+documentation for example whitelisting). 
+
+`kbn-xsrf: true`:: By default, Kibana requires the `kbn-xsrf` header set to true
+for any API interactions.
+
+`Content-Type: application/json`:: The Kibana API uses JSON for the request and
```

Comment:
Since we refer to the Kibana APIs as multiples, I would change this to:

`The {kib} APIs use JSON for request and response bodies. To interact with the APIs, set the `Content-Type` accordingly.`